wvd-templates/DSC/Functions.ps1

```diff
@@ -444,28 +444,32 @@ function InstallRDAgents {
         throw "No registration token specified"
     }
 
-    RunMsiWithRetry -programDisplayName "RDAgentBootLoader" -isUninstall -argumentList @("/x {A38EE409-424D-4A0D-B5B6-5D66F20F62A5}", "/quiet", "/qn", "/norestart", "/passive") -msiOutputLogPath "C:\Users\AgentBootLoaderUnInstall.txt" -msiLogVerboseOutput:$EnableVerboseMsiLogging
-
-    while ($true) {
-        try {
-            $oldAgent = Get-Package -ProviderName msi -Name "Remote Desktop Services Infrastructure Agent"
-        }
-        catch {
-            #Ignore the error if it was due to no packages being found.
-            if ($PSItem.FullyQualifiedErrorId -eq "NoMatchFound,Microsoft.PowerShell.PackageManagement.Cmdlets.GetPackage") {
-                break
+    $msiNamesToUninstall = @(
+        @{ msiName = "Remote Desktop Services Infrastructure Agent"; displayName = "RD Infra Agent"; logPath = "C:\Users\AgentUninstall.txt"}, 

```

Comment:
Maybe change "RD Infra Agent" to "RDInfraAgent" or change "RDAgentBootLoader" to "RD Agent Boot Loader" to make those names follow consistent pattern

Reply:
Those display values are output in the ScriptLog.log and are also included in exception messages, so I'm concerned that changing them could break custom log monitoring or custom code by customers looking for certain exception strings.

Reply:
Thanks for the catching the Configuration.zip! I have updated Configuration.zip in both v1 and v2 templates.

Reply:
Which custom log monitoring are you referring to ? Would it fail the deployment ?

Reply:
For the custom log monitoring that would be something like a customer ingests the ScriptLog.log into their own monitoring system to look for certain strings like "RD Infra Agent". No it wouldn't cause any problems with the ARM template itself.

Reply:
I'm going to merge this to staging for now so I can run the CI. Ping me if you want to discuss this more.